ios/README.md

```diff
@@ -1,4 +1,4 @@
-## MaaS Components for iOS
+## MaaS for iOS
```

Comment:
deja vu, but the SDK is still called maas-components for now 🙂

Reply:
🤦